### DISCUSSION TITLE
How to access information within a sub instance through the ref tag?

### DISCUSSION THREAD

**Discussion by jean-jagnow** [alpinejs/alpine#1011] [4. General]
function test() {
            return {
                modelForm: {
                    label: ""
                }
            }
        }
<div>

    <div x-data="test()">
        <input x-model="modelForm.label" />

        <div x-data="{options:[]}" x-ref="modelForm">
            <label x-label="$refs.modelForm.label"></label>
        </div>
    </div>
</div>

**Comment by HugoDF** [comment]
I don't think you'll be able to successfully use `x-ref`. For example in the case of the `modelForm` ref, as far as Alpine is concerned, it belongs to the nested component and not the one with `x-data="test()"`
